for-jenkins/publishscisoft.sh

```diff
@@ -16,8 +16,8 @@ function publishscisoft(){
 
  ./copyFromJenkins \
 -m SLF7 \
--q s120a-c7  \
--q s120a-e26 \
+-q s112-c7  \
+-q s112-e20 \
```

Comment:
also here for lined 19-20, why are the qualifiers 'older' ?

Reply:
These come from originally pulling from develop and not v1_09_00. This and the other version skew are fixed.

Reply:
@dtorretta56 @cerati @hausnerh Those two files are useed by the Jenkins build system, and I often make adjustments to them before creating the release build. They do not impact the Icarus release quality. Therefore, if those are the only two files in question, I recommend that you approve this request.

Reply:
thanks Gennadiy. Then I approve this request.